docs/spec_draft.md

```diff
@@ -609,6 +610,44 @@ matrix, then the behavior is undefined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.complex
+
+### Semantics
+
+Performs element-wise conversion to a complex value from a pair of real and
+imaginary values, `lhs` and `rhs`, and produces a `result` tensor.
+
+### Inputs
+
+| Name  | Type                          |
+|-------|-------------------------------|
+| `lhs` | tensor of type `f32` or `f64` |
+| `rhs` | tensor of type `f32` or `f64` |
+
+### Outputs
+
+| Name     | Type                   |
+|----------|------------------------|
+| `result` | tensor of complex type |
+
+### Constraints
+
+  * (C1) `lhs` and `rhs` have the same floating-point element type.
```

Comment:
How about element_type(`lhs`) $=$ element_type(`rhs`). As we already mentioned that the element type is floating point in the table.